pkg/apis/contrail/v1alpha1/kubemanager_types.go

```diff
@@ -454,6 +467,10 @@ func (c *Kubemanager) ConfigurationParameters() KubemanagerConfiguration {
 		ipFabricSnat = KubernetesIPFabricSnat
 	}
 
+	if c.Spec.ServiceConfiguration.AuthMode != AuthenticationModeNoAuth && c.Spec.ServiceConfiguration.AuthMode != AuthenticationModeKeystone {
```

Comment:
I don't like it that we check every available mode. What if we'll add more auth modes in the future? That'll make it a pretty long statement. Cannot it be checked somehow different? Maybe enum on struct would ensure that passed vale makes sense or simple check that it's not empty.

Reply:
Changed it to only check if AuthMode is empty string.